ansible_base/rbac/api/views.py

```diff
@@ -257,6 +258,61 @@ class RoleUserAssignmentViewSet(BaseAssignmentViewSet):
         ansible_id_backend.RoleAssignmentFilterBackend,
     ]
 
+    @extend_schema_if_available(
+        request={
+            'application/json': {
+                'schema': {
+                    'allOf': [
+                        {'$ref': '#/components/schemas/RoleUserAssignment'},  # Keep auto-generated schema
+                        {
+                            'oneOf': [
+                                {'required': ['user'], 'not': {'required': ['user_ansible_id']}},
+                                {'required': ['user_ansible_id'], 'not': {'required': ['user']}},
+                            ]
+                        },
+                        {
+                            'oneOf': [
+                                {'properties': {'object_id': {'type': 'integer'}, 'object_ansible_id': False}},
+                                {'properties': {'object_ansible_id': {'type': 'string', 'format': 'uuid'}, 'object_id': False}},
+                                {'not': {'anyOf': [{'required': ['object_id']}, {'required': ['object_ansible_id']}]}},
+                            ]
+                        },
+                    ]
+                }
+            },
+            'application/x-www-form-urlencoded': {
+                'schema': {
+                    'allOf': [
+                        {'$ref': '#/components/schemas/RoleUserAssignment'},
+                        {
+                            'oneOf': [
+                                {'required': ['user'], 'not': {'required': ['user_ansible_id']}},
+                                {'required': ['user_ansible_id'], 'not': {'required': ['user']}},
+                            ]
+                        },
+                    ]
+                }
+            },
+            'multipart/form-data': {
+                'schema': {
+                    'allOf': [
+                        {'$ref': '#/components/schemas/RoleUserAssignment'},
+                        {
+                            'oneOf': [
+                                {'required': ['user'], 'not': {'required': ['user_ansible_id']}},
+                                {'required': ['user_ansible_id'], 'not': {'required': ['user']}},
+                            ]
+                        },
+                    ]
+                }
+            },
+        },
+        description="Create role assignment. Must specify 'role_definition' and exactly one of 'user' or 'user_ansible_id'. "
+        "Can specify at most one of 'object_id' or 'object_ansible_id' (omit both for global roles).",
```

Comment:
```suggestion
        description="Give a user permission to a resource, an organization, or globally (when allowed). Must specify 'role_definition' and exactly one of 'user' or 'user_ansible_id'. "
        "Can specify at most one of 'object_id' or 'object_ansible_id' (omit both for global roles).",
         "The content_type of the role definition and the provided object_id are used to look up the resource.",
         "After creation, the assignment cannot be edited, but can be deleted to remove those permissions."
```

And as per my [prior suggested diff](https://github.com/ansible/django-ansible-base/compare/devel...AlanCoding:django-ansible-base:assignment_repair), please delete the existing docstring. I do not like it when content is duplicated in 2 places.